psydac/linalg/stencil.py

```diff
@@ -27,17 +28,25 @@ def compute_diag_len(pads, shifts_domain, shifts_codomain, return_padding=False)
         shifts_domain : tuple_like (int)
          Shifts of the domain along each direction
 
-        mshifts_codomainc : tuple_like (int)
+        shifts_codomain : tuple_like (int)
          Shifts of the codomain along each direction
 
         return_padding : bool
             Return the new padding if True
-        """
+    
+        Returns
+        -------
+        n : (int)
+         Diagonal lenght of the stencil matrix
+
+        ep : (int)
+          Starting index of the non zero elements 
```

Comment:
Is this the padding? If so please explain it in the docstring

Reply:
yes ,it's the padding, I have added it to the docstring